paperqa/types.py

```diff
@@ -126,6 +126,7 @@ class PQASession(BaseModel):
     id: UUID = Field(default_factory=uuid4)
     question: str
     answer: str = ""
+    answer_reasoning: str | None = None
```

Comment:
Can you add a `description` here just stating that if the model supports reasoning, we include it here